content/flutter-publishing/publishing-to-microsoft-store.md

```diff
@@ -62,6 +62,9 @@ The very first version of the app must be submitted in the Partner Center manual
 
 #### Version your package
 
-To version your `MSIX package` for Microsoft Store Submission, you can either set the value as a string, such as `1.0.0.0`, or use Codemagic's read-only environment variables (`$PROJECT_BUILD_NUMBER`, `$BUILD_NUMBER`) to automatically increment versions on every release.
+Check out how to version your package in the [Microsoft documentation](https://docs.microsoft.com/en-
+us/windows/uwp/publish/package-version-numbering). Note that per Microsoft Store requirements applications are not allowed to have a version with a revision number (last digit of the version) other than zero.
 
-In order to do so, you can simply set the `Package version` field as `1.$BUILD_NUMBER.0.0`, for example, which will use the Codemagic build number for the given workflow to set the package version.
+To version your `MSIX package` for Microsoft Store Submission with Codemagic, you can either set the value as a string, such as `1.0.0.0`, or use Codemagic's read-only environment variables (`$PROJECT_BUILD_NUMBER`, `$BUILD_NUMBER`) to automatically increment versions on every release.
+
+In order to do so, you can simply set the `Package version` field as `1.0.$BUILD_NUMBER.0`, for example, which will use the Codemagic build number for the given workflow to set the package version.
```

Comment:
can you add disclaimer that this is one of many possible approaches to versioning your application and is used only as one example.